src/server/utils/onModuleLoad.js

```diff
@@ -33,21 +33,8 @@ import { setRedirectAllowList } from './redirectAllowList';
 // Trim build hash
 const { buildVersion } = readJsonFile('../../../.build-meta.json');
 const appVersion = buildVersion.slice(0, -9);
-let modulesUsingExternals = new ImmutableSet();
-
-const registerModuleUsingExternals = (moduleName) => {
-  modulesUsingExternals = modulesUsingExternals.add(moduleName);
-};
-
-const clearModulesUsingExternals = () => {
-  modulesUsingExternals = modulesUsingExternals.clear();
-};
 
-export const getModulesUsingExternals = () => modulesUsingExternals.toJS();
-
-export const setModulesUsingExternals = (moduleNames) => {
-  modulesUsingExternals = new ImmutableSet(moduleNames);
-};
```

Comment:
Removing parts of the old system? is this still backwards compatible and this functionality is now achieved in a different way?

Reply:
for the most part this has been reimplemented in holocron - https://github.com/americanexpress/holocron/pull/143/files#diff-c59d732ae8137c4f28a025d9cabd09b4fd3166326f1822eb6b9a94089d89d777R68-R77

it does appear that the call to `clearModulesUsingExternals` has been removed when it should not have.